src/app/feature/vehicles-management/vehicles-management.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="col-lg-12 pt-4">
+  <h2>Vehicles list</h2>
+
+  <div class="mt-4">
+    <p-table [columns]="columns" [value]="vehicles" [paginator]="true" [rows]="10">
+      <ng-template pTemplate="header" let-columns>
+        <tr>
+          <th style="text-align: center" *ngFor="let column of columns" [pSortableColumn]="column.field">
+            {{column.header}}
+            <p-sortIcon [field]="column.field" ariaLabel="Activate to sort" ariaLabelDesc="Activate to sort in descending order" ariaLabelAsc="Activate to sort in ascending order"></p-sortIcon>
+          </th>
+        </tr>
+      </ng-template>
+      <ng-template pTemplate="body" let-vehicle let-columns="columns">
+        <tr >
+
+          <td style="max-width: 650px" *ngFor="let column of columns" [ngClass]="vehicle[column.field] >=0  ? 'is-number' : null">
```

Comment:
Wyciągałbym wszelkie style do cssów, łatwiej to potem wsio zmieniać, bo wszystko masz w jednym miejscu.